GVRf/Framework/framework/src/main/java/org/gearvrf/shaders/GVRPBRShader.java

```diff
@@ -44,7 +44,7 @@
     public GVRPBRShader(GVRContext gvrcontext)
     {
          super("float4 diffuse_color; float4 specular_color; float4 emissive_color; float metallic; float roughness; float specular_exponent; float lightmapStrength; float normalScale; float glossinessFactor",
-                "sampler2D diffuseTexture; sampler2D metallicRoughnessTexture; sampler2D specularTexture; sampler2D lightmapTexture; sampler2D diffuseTexture1; sampler2D normalTexture; sampler2D emissiveTexture",
+                "sampler2D diffuseTexture; sampler2D metallicRoughnessTexture; sampler2D specularTexture; sampler2D lightmapTexture; sampler2D diffuseTexture1; sampler2D normalTexture; sampler2D emissiveTexture; sampler2D brdfLUTTexture; samplerCube diffuseEnvTex; samplerCube specularEnvTexture",
```

Comment:
Binding for texture descriptor starts from 5. Give binding locations for eg: diffuseTexture 5, metallicRoughnessTexture 6 and so on in shader.